src/lib/layouts/DocsArticle.svelte

```diff
@@ -16,11 +16,22 @@
     import { scrollToTop } from '$lib/actions/scrollToTop';
 
     import { Feedback } from '$lib/components';
+    import { onMount } from 'svelte';
 
     export let title: string;
     export let toc: Array<TocItem>;
     export let back: string | undefined = undefined;
     export let date: string | undefined = undefined;
+
+    let reducedArticleSize = false;
+
+    onMount(() => {
+        // reduces size for articles using numbered sections.
+        const slotWrapper = document.querySelector('.slot-wrapper');
+        reducedArticleSize = !!slotWrapper?.querySelector(
+            '.web-article-content-section.is-with-line .web-numeric-badge'
+        );
```

Comment:
can we solve that with svelte context? this doesn't really age well with targetting classnames imo

Reply:
Done.